app/assets/javascripts/static_pages.js

```diff
@@ -1 +1,219 @@
 
+// $(document).ready(function(){
```

Comment:
You said you fixed it? So merge after you fixed it. Then I will pull
request
 On 17 Feb 2015 23:14, "Harry Chen" <notifications@github.com> wrote:

> In app/assets/javascripts/static_pages.js
> <https://github.com/cchchoi1986/hikehk/pull/23#discussion_r24821810>:
>
> > @@ -1 +1,219 @@
> >
> > +// $(document).ready(function(){
>
> What do you mean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cchchoi1986/hikehk/pull/23/files#r24821810>.
>